### ISSUE TITLE
Accessing MVC request context in handlers

### ISSUE THREAD

**Issue by dansoper** [dotnet/AspNetCore.Docs#17330]
I believe this section is no longer true in ASP.NET Core 3.0?
https://github.com/dotnet/aspnetcore/issues/11075

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b1141d92-adeb-6841-229f-914c808f9a2b
* Version Independent ID: 7089b376-949f-c1c6-c37a-5d8cc7ecf104
* Content: [Policy-based authorization in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/security/authorization/policies?view=aspnetcore-3.1#accessing-request-context-in-handlers)
* Content Source: [aspnetcore/security/authorization/policies.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/security/authorization/policies.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-security**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by Rick-Anderson** [comment]
@blowdart  please suggest what in this doc needs to be surrounded by
```
::: moniker range="< aspnetcore-3.0"
::: moniker-end
```

Is it just 

##Accessing MVC request context in handlers

**Comment by Rick-Anderson** [comment]
@serpent5 , isn't this related to [this SO](https://stackoverflow.com/questions/58565574/reading-the-authorizationfiltercontext-in-netcore-api-3-1/58566024#58566024) response?

**Comment by serpent5** [comment]
> 
> 
> @serpent5 , isn't this related to [this SO](https://stackoverflow.com/questions/58565574/reading-the-authorizationfiltercontext-in-netcore-api-3-1/58566024#58566024) response?

Yep, that's right. What I present there appears to be the recommend approach to access `HttpContext` in 3.0+. See https://github.com/dotnet/AspNetCore.Docs/pull/15218 for more.
